src/app/components/booking/partial/booking-form-1.tsx

```diff
@@ -0,0 +1,311 @@
+"use client";
+
+import InfoSvg from "@/app/assets/svgs/InfoSvg";
+import CommonInput from "@/app/components/common/inputs/input";
+import CommonInputFile from "@/app/components/common/inputs/fileInput";
+import { useTranslation } from "react-i18next";
+import React, { FC, useRef, useState } from "react";
+import UploadSvg from "@/app/assets/svgs/UploadSvg"
+import ImageSvg from "@/app/assets/svgs/ImageSvg"
+import Image from "next/image";
+
+const FormWrapper = ({ children }: { children: React.ReactNode }) => {
+  return <div className="relative w-[48%]">{children}</div>
+};
+
+export default function BookingForm1({ data, setData }: { data: any, setData: any }) {
+  const [photoIdFront, setPhotoIdFront] = useState<string>();
+  const [photoIdFrontPreview, setPhotoIdFrontPreview] = useState<string>();
+  const [photoIdFrontSize, setPhotoIdFrontSize] = useState<string>();
+  const [photoIdBack, setPhotoIdBack] = useState<string>();
+  const [photoIdBackPreview, setPhotoIdBackPreview] = useState<string>();
+  const [photoIdBackSize, setPhotoIdBackSize] = useState<string>();
+  const { t } = useTranslation();
+  const IdFront = useRef<any>();
+  const IdBack = useRef<any>();
+  const onClickIdFront = () => {
+    (IdFront.current as any).click();
+  };
+  const onClickIdBack = () => {
+    (IdBack.current as any).click();
+  };
+
+  const onChangeIdFront = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files ? e.target.files[0] : null;
+    if (file) {
+      const fileUrl = URL.createObjectURL(file);
+      const fileSizeInBytes = file.size;
+      const fileSizeInKilobytes: any = (fileSizeInBytes / 1024).toFixed(1);
+      const fileSizeInMegabytes: any = (fileSizeInKilobytes / 1024).toFixed(1);
+      if (fileSizeInMegabytes < 1) {
+        setPhotoIdFrontSize(`${fileSizeInKilobytes} KB`)
+      }
+      setPhotoIdFrontSize(`${fileSizeInMegabytes} MB`)
+      setPhotoIdFrontPreview(fileUrl);
+      setPhotoIdFront(file.name); // Assuming you want to keep the file name
+    }
+  };
+  const onChangeIdBack = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files ? e.target.files[0] : null;
+    if (file) {
+      const fileUrl = URL.createObjectURL(file);
+      const fileSizeInBytes = file.size;
+      const fileSizeInKilobytes: any = (fileSizeInBytes / 1024).toFixed(1);
+      const fileSizeInMegabytes: any = (fileSizeInKilobytes / 1024).toFixed(1);
+      if (fileSizeInMegabytes < 1) {
+        setPhotoIdBackSize(`${fileSizeInKilobytes} KB`)
+      }
+      setPhotoIdBackSize(`${fileSizeInMegabytes} MB`)
+      setPhotoIdBackPreview(fileUrl);
+      setPhotoIdBack(file.name);
+    }
+  };
```

Comment:
This function is duplicated, Im sure you can make it generic passing some function as parameter (and then creating two high level functions for clarity)